src/app/item-feed/ItemFeedSidebar.m.scss

```diff
@@ -27,10 +27,6 @@
   &::-webkit-scrollbar {
     display: none;
   }
-
-  > div {
-    background-color: black;
-  }
```

Comment:
Are you sure we can remove this?

Reply:
Will double-check. 

From what I could tell this was just nested (black) panels inside the item feed (also black)

Reply:
Fairly sure is not needed as it is a duplicated nested background. 

I can't find any instances of the items in the feed requiring a different background from the main feed container (on desktop or mobile)